src/EnumerableSet.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Adapted from: OpenZeppelin Contracts v4.4.1 (utils/structs/EnumerableSet.sol)
```

Comment:
The idea was to remove unused functionality, such as `AddressSet`, since we only use `Bytes32Set`, however there is little to be gained from that and we gain the "burder" of proving it's the same code, so I decided to go with a verbatim copy of the latest version available.